### ISSUE TITLE
ErrorPanel font size inconsistencies

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster-web#125]
The error message has larger text than the dialog body:

![screen shot 2018-09-20 at 11 41 31 am](https://user-images.githubusercontent.com/3766511/45830018-30074a80-bcca-11e8-9cf5-38b9c712bc11.png)

Let's make the error panel have a smaller font:

![screen shot 2018-09-20 at 11 43 14 am](https://user-images.githubusercontent.com/3766511/45830141-7c528a80-bcca-11e8-8fe8-dfe44a6782bc.png)

**Comment by HrushikeshBodas** [comment]
I would like to take up this issue

**Comment by helfi92** [comment]
Go for it :)

**Comment by HrushikeshBodas** [comment]
Is there some Assignment method?

**Comment by helfi92** [comment]
No need for an assignment. Just open a pull-request when you're done. I'm here to help in case you have any questions.

**Comment by HrushikeshBodas** [comment]
I looked at the taskcluster-web Style Guide (https://taskcluster-web-styleguide.netlify.com/#errorpanel) and the "./src/components/ErrorPanel/" is missing in the repository

**Comment by helfi92** [comment]
The `ErrorPanel` component comes from http://github.com/mozilla-frontend-infra/components so you won't find it in the style guide.

**Comment by helfi92** [comment]
https://github.com/mozilla-frontend-infra/components/blob/master/src/components/ErrorPanel/ErrorBox.jsx#L12 is most probably the problem.

**Comment by helfi92** [comment]
We don't want to hardcode a font size.

**Comment by helfi92** [comment]
The solution might be simply to remove this value :)

**Comment by helfi92** [comment]
Feel free to send a pull-request to https://github.com/mozilla-frontend-infra/components.

**Comment by helfi92** [comment]
https://github.com/mozilla-frontend-infra/components/pull/17

**Comment by aimenbatool** [comment]
@helfi92 What about this issue? Is this solved? Links and PRs look confusing.

**Comment by helfi92** [comment]
I agree, the links are confusing. There is a pull-request for this issue in https://github.com/mozilla-frontend-infra/components/pull/19, so I would consider this issue taken.
